src/main/java/seedu/awe/model/expense/Cost.java

```diff
@@ -57,6 +57,34 @@ public Cost subtract(Cost c) {
         return new Cost(String.format("%.2f", result));
     }
 
+    /**
+     * Multiplies the cost by the inputted value c.
+     *
+     * @param c Value to multiply the cost by.
+     * @return Result of the multiplication.
+     */
+    public Cost multiply(double c) {
+        double result = cost * c;
+        if (result < 0) {
+            result = 0;
+        }
+        return new Cost(String.format("%2.f", result));
+    }
+
+    /**
+     * Divides the cost by the inputted value c.
+     *
+     * @param c Value to divide the cost by.
+     * @return Result of the division.
+     */
+    public Cost divide(double c) {
+        double result = cost / c;
+        if (result < 0) {
+            result = 0;
+        }
+        return new Cost(String.format("%.2f", result));
```

Comment:
Remove the formatting to 2 dp
toString method takes care of that from a representation POV anyway.